docs/reference/setup/logging-config.asciidoc

```diff
@@ -212,7 +212,14 @@ logs to roll and compress after 1 GB, and to preserve a maximum of five log
 files (four rolled logs, and the active log).
 
 You can disable it in the `config/log4j2.properties` file by setting the deprecation
-log level to `error`.
+log level to `error` like this:
+```
+...
```

Comment:
I guess the context is clear that this is part of a bigger config. How about we remove the ... ?